RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromProtoTracksEDProducer.cc

```diff
@@ -24,6 +24,7 @@
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include <vector>
+#include <cassert>
```

Comment:
same part as https://github.com/cms-sw/cmssw/pull/8393/files#r26731809
both are in 75X for ~two weeks, as of #7798